mmdeploy/backend/ipu/backend_manager.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Please merge the latest master branch. We have move available check/version check and environments check callback in backend manager.